asciidoctorj-api/src/main/java/org/asciidoctor/Options.java

```diff
@@ -124,10 +127,30 @@ public void setSafe(SafeMode safeMode) {
         this.options.put(SAFE, safeMode.getLevel());
     }
 
+    /**
+     * Keeps track of the file and line number for each parsed block. (Useful for tooling applications where the association between the converted output and the source file is important).
+     * 
+     * @param sourcemap
+     *            value.
+     */
+    public void setSourcemap(boolean sourcemap) {
```

Comment:
Would you mind updating this test to use this setter?
https://github.com/asciidoctor/asciidoctorj/blob/1a33db5703859b5c8ce9aa840270350f65f8e263/asciidoctorj-core/src/test/java/org/asciidoctor/WhenAsciiDocIsRenderedToDocument.java#L263-L268